app.js

```diff
@@ -1,4 +1,87 @@
-// import functions and grab DOM elements
+import { makeStatsString } from './utils.js';
+const castleMenu = document.querySelector('#castle-menu');
+console.log(castleMenu)
+const skylineMenu = document.querySelector('#skyline-menu');
+const waterfrontMenu = document.querySelector('#waterfront-menu');
+const castleImageEl = document.querySelector('#castle-image');
+const skylineImageEl = document.querySelector('#skyline-image');
+const waterfrontImageEl = document.querySelector('#waterfront-image');
+const sloganInput = document.querySelector('#slogan-input');
+const sloganButton = document.querySelector('#slogan-button');
+const sloganList = document.querySelector('.list');
+const countEl = document.querySelector('.count');
+
+let castleCount = 0;
+let skylineCount = 0;
+let waterfrontCount = 0;
+let slogans = [];
+
+castleMenu.addEventListener('change', () => {
+
+    const id = castleMenu.value;
+
+    castleImageEl.src = `./assets/${id}-castle.jpeg`
```

Comment:
There's a bug in this image path that's breaking your code. Check the castle image files and make sure their names are the same as what's being generated here!